myria/test/test_query.py

```diff
@@ -187,6 +188,9 @@ def test_dict(self):
             query = MyriaQuery(COMPLETED_QUERY_ID, connection=self.connection)
             self.assertEqual(query.to_dict(), TUPLES)
 
+            for i in xrange(len(TUPLES) + 1):
```

Comment:
I think this is clearer in the corresponding code in `test_relation.py`:
```
for i in xrange(1, len(TUPLES) + 1):
```
since the first iteration is a no-op.